eclipse-collections-code-generator/src/main/resources/api/block/function/primitivePrimitive2ToPrimitive2Function.stg

```diff
@@ -0,0 +1,40 @@
+import "copyrightAndOthers.stg"
+
+hasTwoPrimitives() ::= "true"
+
+targetPath() ::= "org/eclipse/collections/api/block/function/primitive"
+
+fileName(primitive1, primitive2, sameTwoPrimitives) ::= "<primitive1.name><primitive2.name>To<primitive2.name>Function"
+
+class(primitive1, primitive2, sameTwoPrimitives) ::= <<
+<body(primitive1.type, primitive2.type, primitive1.name, primitive2.name)>
+>>
+
+body(type1, type2, name1, name2) ::= <<
+<copyrightAndOthers()>
+
+package org.eclipse.collections.api.block.function.primitive;
+
+import java.io.Serializable;
+
+/**
+ * This file was automatically generated from template file primitivePrimitiveToPrimitive2Function.stg.
```

Comment:
Why the digit 2 in the name? Shouldn't it just be primitivePrimitiveToPrimitiveFunction?

Reply:
For the record, in the merged PR #312, I named the file `primitivePrimitiveToPrimitive1Function`.

Reply:
@victornoel that was a miss on our side to not ask you to rename. Can you raise another PR with the correct name. Let us fix it before more people start using it. We will merge the updated name first and then this PR.  Now since the other function exists, you will be able to just use that one. Correct?

Reply:
> Now since the other function exists, you will be able to just use that one. Correct?

No, they are two different functions: there is `Type1Type2ToType1` on one hand and `Type1Type2ToType2` on the other hand. So there should be two different files, and thus, the naming